channels/iq.m3u

```diff
@@ -137,3 +137,9 @@ https://601246d26b133.streamlock.net/live/_definst_/1/playlist.m3u8
 https://113483.global.ssl.fastly.net/edge-en/ngrp:live_31b220e0e20611eab1b9cfce247e5d5f_all/index.m3u8
 #EXTINF:-1 tvg-id="AlGhadeer.iq" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/U1TEfvn.png" group-title="News",AlGhadeer (720p)
 https://asdiuhiu12.myvodu.app:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AmozhgaryTV.iq" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/SRotZrm.png" group-title="Religious",Amozhgary TV (1080p)
+https://media.streambrothers.com:1936/8250/8250/playlist.m3u8
+#EXTINF:-1 tvg-id="CihanTV.iq" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/FoDSbvF.png" group-title="General",Cihan TV (720p)
+http://liveshell.net:1935/live/cihann/playlist.m3u8
+#EXTINF:-1 tvg-id="AfarinTV.iq" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/Uw4DX17.png" group-title="Kids",Afarin TV (1080p)
+https://5ab29cc78f681.streamlock.net/afarinTV/livestream/playlist.m3u8
```

Comment:
Link no longer works.
![image](https://user-images.githubusercontent.com/7253922/133007757-ffea1bfa-327b-49d2-9067-aac3a22a4101.png)

Reply:
Dang it! It was just working. Not a reliable link, I removed Afarin for now.